### PR TITLE
Issue 78: Replace deprecated notify()

### LINKED ISSUE #78: Moodle notify() is fully deprecated
`notify()` has been fully deprecated since Moodle 2.0 and needs to be replaced by `$OUTPUT->notification()`

### PR DESCRIPTION
**Pull request by golenkovm**
This resolves #78

### CODE REVIEW

**Comment by PhilippImhof** [comment]
Thank you for your contribution. We will integrate that with the next release.

File: question.php [hunk 1]
```diff
@@ -857,7 +859,7 @@ public function compute_final_grade($responses, $totaltries) {
  * Class to represent a question subpart, loaded from the question_answers table
  * in the database.
  *
- * @copyright  2012 Jean-Michel V�drine
+ * @copyright  2012 Jean-Michel V�drine
```
Comment (PhilippImhof) [inline question.php]:
```suggestion
 * @copyright  2012 Jean-Michel Védrine
```

Comment (PhilippImhof) [reply question.php]: Somehow the é got lost, probably an encoding problem.

**Review by PhilippImhof — CHANGES_REQUESTED** [review]
Looks good to me, thanks for the contribution. Could you please just change back the é in Jean-Michel's name?

File: question.php (hunk 1 above)
Comment (golenkovm) [reply question.php]: Yeah, I think this is encoding thing. My dev env is Phpstorm under Ubuntu and this letter was swapped automatically when I opened the codebase. Tried to undo but Phpstorm didn't allow me to do so. I'm happy to commit your suggested change.

Comment (golenkovm) [reply question.php]: Hmm, looks like the suggested change did nothing. I still can see a letter with question mark in it as an old letter and é as a new value

**Review by PhilippImhof — APPROVED** [review]
Thanks!

**Comment by PhilippImhof** [comment]
Could you please sync your fork and rebase the branch? (It seems I cannot commit to this PR.)

File: question.php (hunk 1 above)
Comment (PhilippImhof) [reply question.php]: For me, it looks good now. I'll have a look at it again, once the branch is rebased.

**Comment by golenkovm** [comment]
Looks to be rosolved now.
Cheers

**Comment by PhilippImhof** [comment]
Perfect. Thanks for your contribution.

**Pull request merged**